src/pages/[scheduleId]/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { MainLayout } from '@/components/layout/MainLayout'
+import { MemoList } from '@/components/screen/MemoList'
+import { USERS, SCHEDULE } from '@/constant/data'
+
+export default function Home() {
+  return (
+    <MainLayout>
+      <div className='m-auto mt-20 w-4/5 md:gap-4 md:grid'>
+        <div className='col-span-2 ml-12 md:ml-28 gap-4'>
+          <div className='gap-4 underline underline-offset-8'>
+            <p className='my-auto'>
+              <span className='text-sm md:text-xl font-semibold'>イベント名　</span>
+              <span className='text-sm md:text-xl font-semibold'>{SCHEDULE.name}</span>
+              <span className='text-xs md:text-sm font-semibold'>参加者数{USERS.length}人</span>
+            </p>
+          </div>
```

Comment:
これはp+spanでやるべき内容ではないかなと思います
p+spanは一つの文章中の一部の文をデコレーションするときに使うべきで、今回spanで囲っている内容はそれぞれ別の文字列なので、全体をdivで囲んでpタグでそれぞれ区切った方が役割がわかりやすいです

Reply:
underlineはtextデコレーションの下線をひくclassで、フォントサイズが違えば下線はバラバラになります
ブロックに対して下線を引きたいのならstrokeを使いましょう

Reply:
gapはflexやgridに付随するもので、
`<div className='gap-4 underline underline-offset-8'>`としてもこのdivはflexでもgridでもないので意味をなさないです、、、
また、flexやgridはその直下の子要素に対して作用するので、今回pタグ一個しか直下にないのでgapは効きません

Reply:
複数の要素を等間隔に並べて中央に寄せたい等の実装はflexを使いましょう

また、フォントの太さや大きさは、見出しかそうでないかでメリハリをつけたらいいんじゃないかと思います
以上を総括して自分なりに直してみましたが、スマホ対応など細かい部分までは実装していないのでそこの対応はよろしくお願いします

```suggestion
          <div className='flex items-center gap-4 border-b-2 border-primary'>
            <div className='bg-primary p-2'>
              <p className='text-xl font-bold text-white'>イベント名</p>
            </div>
            <p className='text-xl'>{SCHEDULE.name}</p>
            <p className='ml-auto text-sm'>参加者数{USERS.length}人</p>
          </div>
```